core/templates/dev/head/pages/exploration_editor/loading_modal_directive.html

```diff
@@ -0,0 +1,30 @@
+<script type="text/ng-template" id="modals/loading">
+  <div class="oppia-loading-body oppia-save-publish-loading">
+    <p>
+      Reloading last saved version<loading-dots></loading-dots>
+    </p>
+  </div>
+</script>
+
+<style>
+  .oppia-loading .modal-content {
+    border-radius: 0;
+    height: 100%;
+    width: 80%;
+  }
+
+  .oppia-loading{
+    top: 50%;
+    left: 50%;
+    height: 200px;
+    border-radius: 10px;
```

Comment:
Why set the `border-radius` .. it doesn't seem to be visible anywhere in the UI ?